### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by zhangerjun**
ADD MY PROJECT FOLDER

### CODE REVIEW

File: content/en/project/BHS_Project_dMRI/index.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+8. Statistical method and machine learning
+
+### Results
+
+#### Data Visualization (see [gif figure](https://github.com/brainhack-school2020/BHS_Project_dMRI/tree/master/Visualization) below):
+  - 3D volume slices image 
+  - Interactive widgets use to show preprocess results 
+  - Interactive widgets use to show reconstruction results 
+<div align="center">
+	<img src="https://github.com/brainhack-school2020/BHS_Project_dMRI/blob/master/Visualization/VolumeSlicesImg.gif" width="300" alt="1" title="3D volume slices image">
```
Comment (pbellec) [inline content/en/project/BHS_Project_dMRI/index.md:110]: this link is broken. To render on the website, you need to add the file to the project folder, and then link like `VolumeSlicesImg.gif`

File: content/en/project/BHS_Project_dMRI/index.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+	<img src="https://github.com/brainhack-school2020/BHS_Project_dMRI/blob/master/Visualization/Preprocessing.gif" width="300" title="Interactive widgets use to show preprocess results">
```
Comment (pbellec) [inline content/en/project/BHS_Project_dMRI/index.md:111]: same comment as above

File: content/en/project/BHS_Project_dMRI/index.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+### Results
+
+#### Data Visualization (see [gif figure](https://github.com/brainhack-school2020/BHS_Project_dMRI/tree/master/Visualization) below):
+  - 3D volume slices image 
+  - Interactive widgets use to show preprocess results 
+  - Interactive widgets use to show reconstruction results 
+<div align="center">
+	<img src="https://github.com/brainhack-school2020/BHS_Project_dMRI/blob/master/Visualization/VolumeSlicesImg.gif" width="300" alt="1" title="3D volume slices image">
+	<img src="https://github.com/brainhack-school2020/BHS_Project_dMRI/blob/master/Visualization/Preprocessing.gif" width="300" title="Interactive widgets use to show preprocess results">	
+	<img src="https://github.com/brainhack-school2020/BHS_Project_dMRI/blob/master/Visualization/ReconstructedImg.gif" width="300" title="Interactive widgets use to show reconstruction results">
```
Comment (pbellec) [inline content/en/project/BHS_Project_dMRI/index.md:112]: same comment as above.

**Comment by pbellec** [comment]
LGTM thank you for the submission!

File: content/en/project/BHS_Project_dMRI/index.md (hunk 3 above)
Comment (zhangerjun) [reply content/en/project/BHS_Project_dMRI/index.md]: > LGTM thank you for the submission!

Thanks, Pierre!

**Pull request merged**